CHANGES.md

```diff
@@ -105,6 +105,10 @@ Release Notes.
 * Activate `receiver-otel#enabledOcRules` receiver with `k8s-cluster,k8s-node,k8s-service,oap,vm` rules on default.
 * Activate `satellite,spring-sleuth` for `agent-analyzer#meterAnalyzerActiveFiles`  on default.
 * Activate `receiver-zabbix` receiver with `agent` rule on default.
+* Replace Http server (GraphQL, agent http protocol) from Jetty with Armeria.
+* [Breaking Change] Remove configuration `restAcceptorPriorityDelta` (env var: `SW_RECEIVER_SHARING_JETTY_DELTA`, `SW_CORE_REST_JETTY_DELTA`).
+* [Breaking Change] Remove configuration `graphql/path` (env var: `SW_QUERY_GRAPHQL_PATH`).
+* [Breaking Change] Remove configuration `restMinThreads` (env var: `SW_CORE_REST_JETTY_MIN_THREADS`, `SW_RECEIVER_SHARING_JETTY_MIN_THREADS`).
```

Comment:
With removing all these, is there any thread number or other key configurations of Armeria server should be exposed?

Reply:
I noticed MAX_THREAD has been kept.

Reply:
Could you make a comparison about size reducing between two binary tars?

Reply:
There is no much reduction in package size, they are about ~132M the same